content/en/docs/reference/using-api/server-side-apply.md

```diff
@@ -96,6 +96,11 @@ becomes available.
 
 A simple example of an object created using Server-Side Apply could look like this:
 
+{{< note >}}
+`kubectl get` omits managed fields by default. `kubectl get` omits managed fields by default.
```

Comment:
The phrase "`kubectl get` omits managed fields by default. " is repeated, suggest to not repeat
```suggestion
`kubectl get` omits managed fields by default.
```